oat-sa/extension-tao-proctoring
# feat: create script to migrate data from kv to amin table

**Description**
Create a script to move delivery extra data from kv_delivery_monitoring table to delivery_monitoring extra_data column
This PR also contains a fix to update extra_data column when it is null
**Ticket**
https://oat-sa.atlassian.net/browse/PR-495
**How to test**
Install sprint 134
Pass couple of delivery_execution (LTI or not) to produce extra_data
Update to ltiProctoring to v9.1.1
Run the script to create extra_column / configure monitoring to use SimpleMonitoring storage
Run the script (use -h for further options)
```
php index.php '\oat\taoProctoring\scripts\tools\MonitoringExtraFieldMigration' -h
```
Data should be moved to delivery_monitoring extra_data column
